tests/requirements-relaxed.txt

```diff
@@ -20,7 +20,7 @@ ansible-pygments==0.1.1
     #   sphinx-ansible-theme
 antsibull-core==2.1.0
     # via antsibull-docs
-antsibull-docs==2.3.1
+antsibull-docs==2.4.0
```

Comment:
This is the change I was waiting for. All the others should not really have an impact anyway, so I'm fine with keeping them.